src/meson.build

```diff
@@ -211,11 +209,25 @@ mangohud_shared_lib = shared_library(
   'MangoHud',
   objects: mangohud_static_lib.extract_all_objects(),
   link_with: mangohud_static_lib,
-  link_args : link_args,
+  link_args : [link_args, '-Wl,--version-script,@0@'.format(join_paths(meson.current_source_dir(), 'mangohud.version'))],
```

Comment:
This is a lot of duplication, should probably be a variable

Reply:
Not sure about how to do that nicer, I have unique files for all libs and mangohub_opengl uses a different directory as its generated at build time. 

Could also break it into several lines like in the example given in this meson issue about improving mesons handling for these.

https://github.com/mesonbuild/meson/issues/3047